packages/tkg-clusterclass-azure/bundle/config/upstream/base.yaml

```diff
@@ -1379,13 +1377,13 @@ spec:
         lun: 0
         nameSuffix: etcddisk
       image:
-        marketplace:
+        id: unknown
       osDisk:
         diskSizeGB: 128
         managedDisk:
           storageAccountType: Premium_LRS
         osType: Linux
-      sshPublicKey: ${AZURE_SSH_PUBLIC_KEY_B64:=""}
```

Comment:
We need a real ssh key base64 value, as the validation webhook will check if the string is a ssh key.